README.md

```diff
@@ -12,60 +12,84 @@ promotion from universe to main.
-| 4. In Progress (any)                    | MIR team ack (and if needed Security Team ack) done, but now needs the Dependency/Seed change to happen to pull it into Main |
-| 5. Fix Committed (any)                  | All of the above done, waiting for an AA (Archive Admin) to promote the packages |
-| 6. Fix Released (any)                   | Case resolved by an AA |
-| 7. Incomplete (any)                     | Questions/Requests were raised for the bug reporter to resolve/clarify |
-| 8. Won\'t Fix (any)                     | Final Nack or given up by the bug reporter |
-| 9. Invalid (any)                        | No Feedback for a while when in incomplete status |
-
-```
-          Answered       +-7----------+     No feedback        +-9-------+
-      +------------------+ Incomplete +------------------------> Invalid |
-      |   old assign can | any        <--------------------+   | any     |
-      |   be kept as-is  +-----^------+                    |   +---------+
-      |                        |    question               |
-      |                        |                           |
-+-1---v-------+  Triage  +-2---+----+ Security needed  +-3-+------------------+
-|  New/Conf   +--------->+ New/Conf +----------------->+ New/Conf             |
-| unassigned  |          | assigned |                  | assigned to Security |
-+-------------+          +-+--+--+--+                  +-+--+--+--------------+
-                           |  |  |    +-8-----------+    |  |  |
-                           |  |  +----> Won't Fix   <----+  |  |
-                           |  |   Nack| any         | Nack  |  |
-                           |  |       +-------------+       |  |
-                           |  |                             |  |
-                           |  |       +-4-----------+       |  |
-                           |  +-------> In Progress <-------+  |
-                           |    Ack   | any         |  Ack     |
-                           |          +-------------+          |
-                           |             |Change that pulls    |
-                           |             |into Main            |
-+-6------------+           |         +-5-v-----------+         |
-| Fix Released |           +---------> Fix Committed <---------+
-| any          <---------------------+ any           |
-+--------------+  AA promotes        +---------------+
+## Process States
+
+This overview aims to clarify the meaning of MIR bug states, address previous 
+confusion, and indicate the next course of action on a MIR bug.
+
+```mermaid
+%% mermaid flowcharts documentation: https://mermaid.js.org/syntax/flowchart.html
+%%{ init: { 'flowchart': { 'curve': 'monotoneY' } } }%%
+flowchart TD 
+    subgraph New["New / Confirmed¹"]
+        Unassigned["<i>1.</i> Unassigned"]
+        Assigned["<i>2.</i> Assigned"]
+        AssignedToSecurity["<i>3.</i> Assigned to Security team"]
+        
+        Unassigned -->|"start triaging"| Assigned
```

Comment:
Now that we are no more so space constrained I think we should consider adding the actions related to those changes.

On the arrows out of 2. and 3. I'd wish we could have a little box that says something like "Adds the full review as comment to the bug"

Or like adding the incoming actions of
"0. Reporter creates a MIR bug following the templates"
Followed by the transition of "subscribe ubuntu-mir to enter the process"
That would then lead into the box that is "1. Unassigned"

Reply:
Thanks, resolved with [4bb6edd](https://github.com/canonical/ubuntu-mir/pull/23/commits/4bb6eddad4f6313289c27386b9ac6ab8bf81ad4e)